services/storage-users/pkg/command/trash_bin.go

```diff
@@ -53,3 +89,402 @@ func PurgeExpiredResources(cfg *config.Config) *cli.Command {
 		},
 	}
 }
+
+func listTrashBinItems(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:      "list",
+		Usage:     "Print a list of all trash-bin items of a space.",
+		ArgsUsage: "['spaceID' required]",
+		Flags:     []cli.Flag{},
+		Before: func(c *cli.Context) error {
+			return configlog.ReturnFatal(parser.ParseConfig(cfg))
+		},
+		Action: func(c *cli.Context) error {
+			log := logging.Configure(cfg.Service.Name, cfg.Log)
+			tp, err := tracing.GetServiceTraceProvider(cfg.Tracing, cfg.Service.Name)
+			if err != nil {
+				return err
+			}
+			var spaceID string
+			if c.NArg() > 0 {
+				spaceID = c.Args().Get(0)
+			}
+			if spaceID == "" {
+				_ = cli.ShowSubcommandHelp(c)
+				return fmt.Errorf("spaceID is requered")
+			}
+			fmt.Printf("Getting trash-bin items for spaceID: '%s' ...\n", spaceID)
```

Comment:
use `log.Info()` instead of `fmt.Printf`

Reply:
I used the fmt because it should always be visible.

Reply:
I disagree. You can set to `Info` level if you want all the noise. The line is not necessarily needed imho